enterprise/views.py

```diff
@@ -606,11 +607,21 @@ def _enroll_learner_in_course(
                                 course_mode=course_mode
                             )
                         )
-                    except Exception as exc:  # pylint: disable=broad-except
-                        LOGGER.error(
-                            'Unable to create an LMS enrollment from the DSC view: {exc}'.format(
-                                exc=exc
-                            )
+                    except HttpClientError as exc:
+                        monitoring.record_exception()
```

Comment:
I think this is my first time seeing `monitoring.record_exception` being used. Where does this exception get recorded?

Reply:
New Relic!

Reply:
https://github.com/edx/edx-django-utils/blob/10df5a3cfcb94842edab6de48901a57cafbfd11d/edx_django_utils/monitoring/internal/utils.py#L85-L98

>    Records a caught exception to the monitoring system.    Note: By default, only unhandled exceptions are monitored. This function    can be called to record exceptions as monitored errors, even if you handle    the exception gracefully from a user perspective.